morsecode.rb

```diff
@@ -0,0 +1,51 @@
+@dict = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}
+
+def get_letter(letter)
+  @current_letter = @dict[letter]
+  @current_letter
+end
+
+def decode_word(word)
+  @letters = word.split
+  @current_word = ''
+  @letters.each do |letter|
+    get_letter(letter)
+    @current_word += get_letter(letter)
+  end
+  print @current_word
+end
+
+def decode_message(message)
+  @words = message.split('   ')
+  @words.each do |word|
+    decode_word(word)
+  end
+end
```

Comment:
- [ ] Kindly make necessary adjustments to this method to return the decoded sentence accordingly based on the changes made to the `decode_word` method above. Here is an idea:

  ```
  def decode_message(message)
    @words = message.split('   ')
    decoded_sentence = ''
    @words.each do |word|
      decoded_sentence += "#{decode_word(word) + ' '}"
    end
    decoded_sentence.strip
  end
  ```